guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode.adoc

```diff
@@ -0,0 +1,48 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-push-mode_{context}"]
+ifeval::["{context}" == "managing-hosts"]
+= Setting an alternative directory for remote execution jobs in push mode
+
+By default, {Project} uses the `/var/tmp` directory on hosts for remote execution jobs in push mode.
+
+If the `/var/tmp` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
+Use this procedure to set a new directory for executing remote execution jobs in push mode.
+
+:rex-default-working-dir: /var/tmp
+:installer-change-working-dir-opt: --foreman-proxy-plugin-remote-execution-script-remote-working-dir
+endif::[]
+ifeval::["{context}" == "ansible"]
+= Configuring {Project} to use an alternative directory to execute remote jobs on hosts
+
+Ansible puts its own files it requires into the `$HOME/.ansible/tmp` directory, where `$HOME` is the home directory of the remote user.
+You can configure {Project} to use an alternative directory if required.
+
+:rex-default-working-dir: $HOME/.ansible/tmp
+:installer-change-working-dir-opt: --foreman-proxy-plugin-ansible-working-dir
```

Comment:
I'm sorry, I must have taken a wrong turn somewhere when describing how this works for ansible.

The remote directory is indeed `$HOME/.ansible/tmp`, but that's ansible's default and we do not override it or expose it in any way. Per https://docs.ansible.com/ansible/latest/collections/ansible/builtin/sh_shell.html#parameter-remote_tmp one could change that by configuring `remote_tmp` in `defaults` section in `/etc/ansible.cfg` if needed.

`--foreman-proxy-plugin-ansible-working-dir` controls the local working directory (ie on the smart proxy) and seems to be set by default to `/tmp` by us https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/ansible/params.pp#L9

Reply:
I see. That would mean the existing procedure is incorrect to begin with. I updated the procedure for Ansible, see if it is correct now.